vercel/resource_project.go

```diff
@@ -73,6 +73,11 @@ terraform to your Deployment.
 						},
 						Required: true,
 					},
+					"git_branch": {
+						Description: "The git branch of the environment variable.",
+						Type:        types.StringType,
+						Optional:    true,
+					},
```

Comment:
I think we're also missing a test that includes the `git_branch` being used.

Reply:
I think at the moment the git tests only actually work for you. I get ```
Could not create project, unexpected error: bad_request - To link a GitHub
        repository, you need to install the GitHub integration first.
```
Which I think is actually because I can only use my own repositories. However I have added some reasonable tests that will hopefully work.

Reply:
Ahh yeah, makes sense. We should probably read an environment variable for the non-team git repository or something. I don't think we can keep it pointing at my git repos forever 🤣. One for later though!